src/main/java/edu/oregonstate/cs361/battleship/Player.java

```diff
@@ -5,14 +5,22 @@
  */
 
 public class Player {
+  
+    public Player(){
+        Firsthit = new Coordinate(11,11);
+        Lasthit = new Coordinate(11,11);
+        ShipCount = 0;
+        shootstate = 0;
+    }
 
+    public Coordinate Firsthit;
 
-    public coord Firsthit;
-
-    public coord Lasthit;
+    public Coordinate Lasthit;
```

Comment:
Thank you for correcting my mistakes there. The code supposed to be "public Coordinate", but I mistakenly type in "coord"